CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ All notable changes to this project will be documented in this file.
 -   #1853 : Fix translation of a file whose name conflicts with Fortran keywords.
 -   Link and mention `devel` branch, not `master`.
 -   #1047 : Print the value of an unrecognised constant.
+-   #1892 : Fix implementation of list function when list or tuple is passed as parameter.
```

Comment:
I think this solution is more general than you suggest here. E.g. it should also work for sets. Maybe just mention iterables?
```suggestion
-   #1892 : Fix implementation of list function when an iterable is passed as parameter.
```